CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+
+## Your First Contribution
+
+Here are a couple of friendly tutorials with more information about contributing to OSS projects: http://makeapullrequest.com/,  http://www.firsttimersonly.com/, and [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first :smile_cat:
+
+If a maintainer asks you to "rebase" your PR, they're saying that a lot of code has changed, and that you need to update your branch so it's easier to merge. If you'd like to take this opportunity to squash commits into logical, meaningful chunks, please do so, but don't feel obligated to always squash everything down to one commit.
+
+### Run the following commands to get this project set up locally.
+
+```ssh
+# Clone the repo
+$ git clone https://github.com/keenlabs/keen-sdk-net.git && cd keen-sdk-net
+```
+
+Now, launch Visual Studio and open `Keen.sln`. Build the solution, discover unit tests, and run them. Set some breakpoints and get acquainted with the PCL, the .NET 4.5 DLL and the .NET 3.5 DLL, as well as the related test projects. Maybe add a mini test project that references one of the built binaries and play with actually using the SDK as per the instructions in the [README](https://github.com/keenlabs/keen-sdk-net/blob/master/README.md).
+
+### Submitting a Pull Request
+
+When you go about creating a pull request, first make sure you are working in a branch, and not trying to go directly in `master`. Explain what features this PR is adding or bugs this is fixing, and reference the related Issue(s). Make sure some tests are included or at least explain why if they are not. If certain testing steps are not relevant, specify that. If additional manual checks are needed, explain that as well. Please run through all testing steps before asking for a review. Until CI and required checks are in place, tests must be run manually so do your due diligence so that things go as smoothly as possible.
+
+A project maintainer will respond in a few days to the PR, and you can iterate with them and/or other community members until the PR is satisfactory and a project maintainer will go about approving, merging and releasing, and thanking you for your contribution.
+
+## How to report a bug
+If you find a security vulnerability, do NOT open an issue. Email team@keen.io instead.
+
+If you find a bug that's not a security vulnerability please head over to the issues tab of this repo and open up an issue.
+
+## Suggesting features
+
+We welcome your feedback and requests. If you have a straight forward request please open up an issue that details the request. If you want to talk to someone on the Keen team head over to http://keen.chat or send a note to team@keen.io and we will make sure and get you in touch with the appropriate people.
```

Comment:
Yeah I corrected that in another place where I copy/pasted, but missed this one. Will fix it up!